### ISSUE TITLE
Error loading from Git Bash seems to be dup of #58950 from 2018

### ISSUE THREAD

**Issue by DavidLangworthy** [microsoft/vscode#138170]
Does this issue occur when all extensions are disabled?: Yes

Steps to Reproduce:

1. Start code from Git Bash

I have updated both vscode and Git Bash

This is the same behavior as in https://github.com/microsoft/vscode/issues/58950

```
$ code .
internal/modules/cjs/loader.js:937
  throw err;
  ^

Error: Cannot find module 'C:\Program Files\Microsoft VS Code\resources\app\out\cli.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:934:15)
    at Module._load (internal/modules/cjs/loader.js:779:27)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12913)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```
The file is in AppData
```
$ find /c -name cli.js
[...]
/c/Users/me/AppData/Local/Programs/Microsoft VS Code/resources/app/out/cli.js
/c/Users/me/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/code/node/cli.js
```

**Comment by lszomoru** [comment]
@aeschli, any idea what could be the issue here since you have worked on a similar issue in the past?

**Comment by aeschli** [comment]
`C:\Program Files\Microsoft VS Code` is the [vscode system install location](https://code.visualstudio.com/docs/setup/windows#_user-setup-versus-system-setup) but it seems VS Code is now installed as a user install.
Maybe `code` is an alias that is outdated?

**Comment by jayallen** [comment]
Yep. Can confirm.  It exists but it's in the user install location.  Note that my install came from the Microsoft Store.

````
> ls "c:/Users/USERNAME/AppData/Local/Programs/Microsoft VS Code/resources/app/out/vs/code/node/cli.js"

    Directory: C:\Users\USERNAME\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\node


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a----         4/11/2022   7:46 AM         216067 cli.js


> ls 'C:\Program Files\Microsoft VS Code\resources\app\'


    Directory: C:\Program Files\Microsoft VS Code\resources\app


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----         2/11/2022   9:45 AM                licenses
d-----         2/11/2022   9:45 AM                resources
-a----          2/9/2022  10:02 PM         168455 LICENSE.rtf
-a----          2/9/2022  10:02 PM        5369386 LICENSES.chromium.html
-a----          2/9/2022  10:02 PM           9312 package.json
-a----          2/9/2022  10:02 PM         104610 telemetry-core.json
-a----          2/9/2022  10:02 PM          86730 telemetry-extensions.json
-a----          2/9/2022  10:02 PM         946867 ThirdPartyNotices.txt


> ls 'C:\Program Files\Microsoft VS Code\resources\app\resources\'


    Directory: C:\Program Files\Microsoft VS Code\resources\app\resources


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----         2/11/2022   9:45 AM                win32


> ls 'C:\Program Files\Microsoft VS Code\resources\app\resources\win32\'


    Directory: C:\Program Files\Microsoft VS Code\resources\app\resources\win32


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a----          2/9/2022  10:02 PM         154270 bower.ico
-a----          2/9/2022  10:02 PM         151891 c.ico
-a----          2/9/2022  10:02 PM           2069 code_150x150.png
-a----          2/9/2022  10:02 PM           1341 code_70x70.png
-a----          2/9/2022  10:02 PM         152749 config.ico
-a----          2/9/2022  10:02 PM         152144 cpp.ico
-a----          2/9/2022  10:02 PM         152526 csharp.ico
-a----          2/9/2022  10:02 PM         150975 css.ico
-a----          2/9/2022  10:02 PM         149932 default.ico
-a----          2/9/2022  10:02 PM         152187 go.ico
-a----          2/9/2022  10:02 PM         152224 html.ico
-a----          2/9/2022  10:02 PM         152284 jade.ico
-a----          2/9/2022  10:02 PM         153815 java.ico
-a----          2/9/2022  10:02 PM         152092 javascript.ico
-a----          2/9/2022  10:02 PM         152075 json.ico
-a----          2/9/2022  10:02 PM         152207 less.ico
-a----          2/9/2022  10:02 PM         150818 markdown.ico
-a----          2/9/2022  10:02 PM         152538 php.ico
-a----          2/9/2022  10:02 PM         151218 powershell.ico
-a----          2/9/2022  10:02 PM         151915 python.ico
-a----          2/9/2022  10:02 PM         155513 react.ico
-a----          2/9/2022  10:02 PM         150691 ruby.ico
-a----          2/9/2022  10:02 PM         153380 sass.ico
-a----          2/9/2022  10:02 PM         149691 shell.ico
-a----          2/9/2022  10:02 PM         151477 sql.ico
-a----          2/9/2022  10:02 PM         152020 typescript.ico
-a----          2/9/2022  10:02 PM         152224 vue.ico
-a----          2/9/2022  10:02 PM         152974 xml.ico
-a----          2/9/2022  10:02 PM         151113 yaml.ico
````

**Comment by jayallen** [comment]
Note: I use Powershell so that's what you're seeing above.  AND, I figured out that this was a `PATH` environment variable problem.  

Looking at my environment variables, `%USERPROFILE%\AppData\Local\Programs\Microsoft VS Code\bin` was last in my user-level `PATH` list while  `C:\Program Files\Microsoft VS Code\bin` was somewhere near the middle of my System-level `PATH` list.  When the two were combined together, the system PATH list came first.

When I added `%USERPROFILE%\AppData\Local\Programs\Microsoft VS Code\bin` to the top of my System-level `PATH` list and restarted, `code` works again.  I didn't try simply mutating the PATH value from the command line but I'm sure that would work as an easy test.
